pages/create.tsx

```diff
@@ -438,21 +410,21 @@ const CreatePage: NextPage = observer(() => {
             },
           })
         );
+
+        await store.sdk.models.createCpmmMarketAndDeployAssets(params);
       });
     };
 
-  const createScalarMarket = async (): Promise<number> => {
-    const params = await getCreateMarketParameters();
+  const createMarket = async () => {
+    if (!form.isValid) {
+      return;
+    }
 
-    if (isRangeOutcomeEntry(formData.outcomes.value)) {
-      const bounds = [
-        formData.outcomes.value.minimum,
-        formData.outcomes.value.maximum,
-      ];
-      return new Promise(async (resolve, reject) => {
-        await store.sdk.models.createScalarMarket(
-          ...params,
-          bounds,
+    let marketId: number;
```

Comment:
Think this can be declared using `const` a couple of lines below where it's assigned

Reply:
The assignment is done based on a if else.
```ts
let marketId: number;

    if (!deployPool) {
      marketId = await new Promise(async (resolve, reject) => {
        ...
        return parseInt(await store.sdk.models.createMarket(params));
      });
    } else {
      marketId = await createCategoricalCpmmMarketAndDeployPoolTransaction();
    }
```

Reply:
Ah sorry, I must have missed the line expand thing